### PR TITLE
Create luxgen_s5_2014.dbc

### PR DESCRIPTION
**Pull request by chinlin0924**

### CODE REVIEW

File: luxgen_s5_2015.dbc [hunk 1]
```diff
@@ -0,0 +1,155 @@
...
+ SG_ _DOOR_RR_STATUS : 19|1@0+ (1,0) [0|1] "" XXX
+ SG_ _DOOR_BACK_DOOR_STATUS : 22|1@0+ (1,0) [0|1] "" XXX
+ SG_ LEFT_SIGNAL_STATUS : 10|1@0+ (1,0) [0|1] "" XXX
+ SG_ RIGHT_SIGNAL_STATUS : 9|1@0+ (1,0) [0|1] "" XXX
+
+BO_ 832 GEAR_RPM_SPEED_STATUS: 8 XXX
+ SG_ TRANS_MODE : 8|8@1+ (1,0) [0|0] "" XXX
+ SG_ TRANS_GEAR_POS : 0|3@1+ (1,0) [0|1] "" XXX
+ SG_ ENGINE_RPM1 : 55|8@0+ (1,0) [0|255] "" XXX
+ SG_ _ENGINE_TEMP : 63|8@0+ (1,0) [0|255] "KM" XXX
```
Comment (rbiasini) [inline luxgen_s5_2015.dbc]: KM is a temp unit?

File: luxgen_s5_2015.dbc [hunk 2]
```diff
@@ -0,0 +1,155 @@
...
+
+
+
+CM_ SG_ 1104 DRIVER_SEAT_BELT_ONOFF "0 - on , 1 = off";
+CM_ SG_ 1056 _DOOR_RL_STATUS "04 - RL - open";
+CM_ SG_ 1056 _DOOR_FL_STATUS "28 - FL open , 38 - FR";
+CM_ SG_ 1056 RIGHT_SIGNAL_STATUS "R,L shows at same time means hazard";
+CM_ SG_ 832 TRANS_MODE "AT - 85 / MT - 8D";
+CM_ SG_ 832 TRANS_GEAR_POS "R-7 , 0 - N";
+CM_ SG_ 821 CRUSE_ONOFF "定速開關（但未啟動)";
```
Comment (rbiasini) [inline luxgen_s5_2015.dbc]: not sure if cabana will like Chinese characters. Would you mind to translate in english?

Comment (chinlin0924) [reply luxgen_s5_2015.dbc]: sure.. no problem.. that is for memo purpose

File: luxgen_s5_2015.dbc (hunk 1 above)
Comment (chinlin0924) [reply luxgen_s5_2015.dbc]: typo

**Comment by chinlin0924** [comment]
@rbiasini can you help to review my change?
thank you very much

**Comment by chinlin0924** [comment]
hi @rbiasini do you have any extra comments?
thanks

File: luxgen_s5_2015.dbc [hunk 3]
```diff
@@ -0,0 +1,155 @@
...
+
+BO_ 1104 SEATBELT_STATUS: 8 XXX
+ SG_ DRIVER_SEAT_BELT_ONOFF : 21|1@0+ (1,0) [0|3] "" XXX
+
+BO_ 1056 BODY_ECU_STATUS: 8 XXX
+ SG_ _DOOR_RL_STATUS : 18|1@1+ (1,0) [0|255] "" XXX
+ SG_ _DOOR_FL_STATUS : 13|1@0+ (1,0) [0|1] "" XXX
+ SG_ _DOOR_FR_STATUS : 12|1@0+ (1,0) [0|1] "" XXX
+ SG_ _DOOR_RR_STATUS : 19|1@0+ (1,0) [0|1] "" XXX
+ SG_ _DOOR_BACK_DOOR_STATUS : 22|1@0+ (1,0) [0|1] "" XXX
```
Comment (rbiasini) [inline luxgen_s5_2015.dbc]: remove `_` from the beginning of the signal names

File: luxgen_s5_2015.dbc [hunk 4]
```diff
@@ -0,0 +1,155 @@
...
+ SG_ LEFT_SIGNAL_STATUS : 10|1@0+ (1,0) [0|1] "" XXX
+ SG_ RIGHT_SIGNAL_STATUS : 9|1@0+ (1,0) [0|1] "" XXX
+
+BO_ 832 GEAR_RPM_SPEED_STATUS: 8 XXX
+ SG_ TRANS_MODE : 8|8@1+ (1,0) [0|0] "" XXX
+ SG_ TRANS_GEAR_POS : 0|3@1+ (1,0) [0|1] "" XXX
+ SG_ ENGINE_RPM1 : 55|8@0+ (1,0) [0|255] "" XXX
+ SG_ _ENGINE_TEMP : 63|8@0+ (1,0) [0|255] "engine value(not sure)" XXX
```
Comment (rbiasini) [inline luxgen_s5_2015.dbc]: put comments at the end

**Comment by rbiasini** [comment]
@chinlin0924, Can you remove the `.` from the file name?

**Comment by pd0wm** [comment]
There is still a message with underscores in the name. Can you also remove the messages without any signals?

**Comment by chinlin0924** [comment]
> messages

hi, I was told to remove ".".
so I removed the "." in file name.
I am a bit confused, this time is to remove "_" in file name. it looks like a misunderstanding?
can you directly tell me what file name should it be?
and how do you define the messages that don't have any signal?
thank you

**Pull request merged**